helm/cluster-openstack/templates/_openstack_cluster_template.tpl

```diff
@@ -9,6 +9,10 @@ metadata:
 spec:
   template:
     spec:
+      {{- if .Values.managementCluster }}
+      tags:
+      - {{ .Values.managementCluster }}_{{ include "resource.default.name" $ }}
```

Comment:
Isn't it a good idea to prefix it with some known literal? Like `giantswarm.io/cluster={{ .Values.managementCluster }}_{{ include "resource.default.name" $ }}`

Reply:
It is not allowed to use `.` or `/` in the tags section. Those break cluster creation.  We can pass something like `giant_swarm_cluster_`.  Note that we need to change this PR as well: https://github.com/giantswarm/default-apps-openstack/pull/20/files

Reply:
`giant_swarm_cluster_ ` would do for me then. Is there a length limit?

Reply:
Yes. The whole tag has to be shorter than 255.  Much more than our need : ) I will go with `giant_swarm_cluster_`